### PR TITLE
Package bump

### PR DESCRIPTION
**Pull request by ismailToyran**
### Project organization
- Related [trello card](https://trello.com/c/3ehiKkRF/917-update-frontend-libraries-nextjs-and-chakra)

### Description
Bump packages to get the most recent versions. Mainly for React, Next.js and Chakra UI

### TODO
- [x] Fix `useSigner.ts` hook to be compatible with the most recent update
- [x] Update chat repository to be compatible with new Chakra + React version so we can reapply theming and enable chat page back (required: https://github.com/liteflow-labs/chat/pull/32)
- [x] Update remaining 3 packages if needed, @nft/api-graphql, @nft/webhook

### Checklist
- [x] Base branch of the PR is `dev`

### Nice to have but not on this PR
- Updating `next-translate` (had issues, so postponed for now as it's still working as intended)

### CODE REVIEW

File: components/Select/Select.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ type IProps<T extends any> = HTMLAttributes<any> & {
   sortAlphabetically?: boolean
 }
```
Comment (ismailToyran) [inline components/Select/Select.tsx:45]: To be checked

File: connectors.ts [hunk 2]
```diff
@@ -47,16 +35,11 @@ const getDefaultWallets = ({
       groupName: 'Popular',
       wallets: [
```
Comment (ismailToyran) [inline connectors.ts:49]: Double check `projectId` usage

File: hooks/useAccount.ts [hunk 3]
```diff
@@ -24,6 +23,18 @@ const COOKIE_OPTIONS = {
...
+  constructor(public wallet: WalletClient) {}
+
+  async getAddress() {
+    return this.wallet.account.address
+  }
+
+  signMessage(message: string) {
+    return this.wallet.signMessage({ message, account: this.wallet.account })
+  }
+}
```
Comment (antho1404) [inline hooks/useAccount.ts:36]: Let's remove this hack and use the `walletClientToSigner` function from the `useSigner`

File: components/Collection/CollectionHeader.tsx [hunk 4]
```diff
@@ -141,12 +141,11 @@ const CollectionHeader: FC<Props> = ({ collection, loading, reportEmail }) => {
           <Image
             src={collection.cover}
             alt={collection?.name || ''}
-            layout="fill"
-            objectFit="cover"
-            borderRadius={{ base: 0, sm: '2xl' }}
+            fill
```
[the remark this answers is not in the document]
Comment (ismailToyran) [reply components/Collection/CollectionHeader.tsx:144]: [Collection banner missing objectFit added](https://github.com/liteflow-labs/starter-kit/pull/330/commits/c877dbbc763cf9872acbdd1fe164a499163f1f35)
Nice catch, double checked. This was the only one missing for images 👍

File: connectors.ts (hunk 2 above)
Comment (antho1404) [reply connectors.ts:51]: The usage is good, wallet connect is indeed required for all these wallets which is not great but I guess no choice

File: hooks/useAccount.ts [hunk 5]
```diff
@@ -25,14 +25,7 @@ const COOKIE_OPTIONS = {
 }
 
 export default function useAccount(): AccountDetail {
-  const { isConnected, isReconnecting } = useWagmiAccount({
-    // FIXME: Implements dummy onConnect and onDisconnect functions to prevent a bug only present with React 17 where other onConnect and onDisconnect in the same component (eg: AccountProvider) are not triggered if another useWagmiAccount doesn't implement those functions.
-    // See https://github.com/liteflow-labs/starter-kit/pull/230#issuecomment-1477409307 for more info.
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
-    onConnect: () => {},
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
-    onDisconnect: () => {},
```
Comment (antho1404) [inline hooks/useAccount.ts:34]: Great, just FYI it was some discussion resolved about it here https://github.com/wagmi-dev/wagmi/issues/1998

File: pages/users/[id]/bids/placed.tsx [hunk 6]
```diff
@@ -201,11 +201,10 @@ const BidPlacedPage: NextPage<Props> = ({ now }) => {
                             alt={item.asset.name}
                             width={40}
                             height={40}
-                            layout="fixed"
-                            objectFit="cover"
-                            rounded="full"
                             h={10}
                             w={10}
+                            objectFit="cover"
+                            rounded="2xl"
```
[the remark this answers is not in the document]
Comment (antho1404) [reply pages/users/[id]/bids/placed.tsx:207]: Still this issue, please let me know if that's normal

**Review by antho1404 — APPROVED** [review]
Just one last comment about some images and I pushed a commit to improve types but other than that I didn't find any issue, it even kind of fixed an issue (https://github.com/liteflow-labs/starter-kit/issues/301) that we had when refreshing from the nft detail page while logged in where the loader was infinite.

So overall great jobs :)

**Comment by ismailToyran** [comment]
We were using full rounded for only users. This was the only place we were using full rounded for assets as well. (we had them not full rounded in the past) To make it aligned with the rest of the assets, I've made this change. 

Could revert if you want to keep it as it's though.

File: pages/users/[id]/bids/placed.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (ismailToyran) [reply pages/users/[id]/bids/placed.tsx:207]: This is a go from Manu, and he was actually happy with that one 👍

File: components/Explore.tsx [hunk 7]
```diff
@@ -28,36 +28,37 @@ const ExploreTemplate: FC<{
...
-          <Link href={`/explore${searchParam}`} whiteSpace="nowrap" mr={4}>
-            <Tab borderColor="gray.200" pb={4} color="gray.500">
-              <Text as="span" variant="subtitle1">
-                {t('explore.tabs.nfts')}
-              </Text>
-            </Tab>
-          </Link>
-          <Link
+        <TabList gap={4}>
+          <Tab
```
[the remark this answers is not in the document]
Comment (ismailToyran) [reply components/Explore.tsx:32]: Same comment as other Tab component, ok for Manu 👍

**Pull request merged**